config/manifests/bases/ibm-common-service-operator.clusterserviceversion.yaml

```diff
@@ -467,7 +481,7 @@ metadata:
               productID: 068a62892a1e4db39641342e592daa25
               productMetric: FREE
               productName: IBM Cloud Platform Common Services
-              productVersion: 3.4.0
+              productVersion: 3.5.0
```

Comment:
@DanielXLee ProductVersion would be removed in 3Q.

Reply:
okay